tests/e2e/systemtest/mon/test_restore_ceph_mon_quorum.py

```diff
@@ -0,0 +1,135 @@
+import logging
+
+import pytest
+
+from ocs_ci.framework.testlib import E2ETest, ignore_leftovers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pod import wait_for_storage_pods, get_ceph_tools_pod
+from ocs_ci.helpers.helpers import (
+    mon_quorom_lost,
+    recover_mon_quorum,
+    modify_deployment_replica_count,
+)
+from ocs_ci.helpers.sanity_helpers import Sanity
+
+
+log = logging.getLogger(__name__)
+
+
+@ignore_leftovers
+class TestRestoreCephMonQuorum(E2ETest):
+    """
```

Comment:
please check if https://bugzilla.redhat.com/show_bug.cgi?id=1944513 can be covered here.

Reply:
I'll take it up in another PR when nodes are down. We don't see the alerts in this case not sure why? and also it is not recommended to remove the directory directly and fixing the issue will be less priority.